wesbos/Syntax
# Fix missing sponsor image from Instagram

## What's the changes here?

- These changes address a bug where an image hotlinked from Instagram was not loading on the `/sponsor` page.
- I don't know what screenshot was previously used as it's now 403'in so I instead chose another relevant image from [Dhanish's](https://www.instagram.com/dhanishgajjar/) feed.
- I pulled this image and also the other Instagram image used further up the page to the local `static` folder so this page doesn't rely on the whims of Instagram permissions to serve up images anymore.
- I resized the two images to 400px as specified in their image width attributes as they were previously being served at over double that size - `947px x 947px`
- When I simply swapped out the broken image with the new one, the layout was still broken ([screenshot](https://cl.ly/a32744068fe5))? Maybe this was a bug with the previous image also? 🤷🏻‍♂️ Anyways, moving the `<figure>` element up the flow to just after the heading fixed it up.

## How to QA these changes

- Pull this feature branch
- Run `npm install`
- ...followed by `npm run dev` 
- Fire up http://localhost:6969/sponsor
- Check the two Instagram images on the page load and display nicely with the text 👌🏻  

## Screenshot with these changes applied

![localhost_6969_sponsor_-_2018-09-09_22 58 28](https://user-images.githubusercontent.com/505570/45269315-ec1f7480-b483-11e8-8615-0464146cc2dd.png)

Fixes #145
